cg/store/models.py

```diff
@@ -420,6 +420,7 @@ class Invoice(Model):
     excel_kth = Column(types.BLOB)
     excel_ki = Column(types.BLOB)
     price = Column(types.Integer)
+    record_type = Column(types.Text)
 
```

Comment:
I added record_type to the invoice-table in the cg-database. Reflected here in the invice model